UIKitDZ/UIKitDZ/ChangeLabelViewController/ChangeLabelViewController.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  ChangeLabelViewController.swift
+//  UIKitDZ
+//
+//  Created by Анна Сычева on 21.09.2022.
+//
+
+import UIKit
+/// Экран обработки текста в лейбле
+final class ChangeLabelViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let nameColorLabel = "Выберите цвет"
```

Comment:
Label 
```suggestion
        static let nameColorLabel = "Выберите цвет"
```
это разве Label?)